tests/util.py

```diff
@@ -5,6 +5,19 @@
 import textwrap
 import soupsieve as sv
 import sys
+import pytest
+
+try:
+    from bs4.builder import HTML5TreeBuilder  # noqa
```

Comment:
We should probably specify the line error we are ignoring instead of ignoring all lint issues.